liquidator/index.js

```diff
@@ -84,7 +84,7 @@ async function run(address, shouldPoll, pollingDelay, priceFeedConfig, monitorPo
       });
       Logger.info({
         at: "liquidator#index",
```

Comment:
Should capitalize this to `Liquidator` as you did above

Reply:
also good catch. I implemented this change throughout.